src/axom/sidre/tests/sidre_group.cpp

```diff
@@ -1808,7 +1808,7 @@ TEST(sidre_group, save_restore_empty_datastore)
   for(const auto& protocol : protocols)
   {
     const std::string file_path = file_path_base + protocol;
-    ds1->getRoot()->save(file_path, protocol);
+    EXPECT_TRUE(ds1->getRoot()->save(file_path, protocol));
```

Comment:
Are there any cases that test the `EXPECT_FALSE` path?
E.g. by setting `slic::disableAbortOnError()`

https://github.com/LLNL/axom/blob/65dfc3ab4b15ee3208d11eb57ee36e1dcc5a00d9/src/axom/slic/interface/slic.hpp#L133-L140

Reply:
There are no cases that test `EXPECT_FALSE`, at this point.  I should, indeed, try that.

Reply:
Added tests that use `EXPECT_FALSE`.